### PR TITLE
[4.0] Shorten user list headlines

### PR DESCRIPTION
**Pull request by bembelimen**
Pull Request for Issue # .

### Summary of Changes
Shorten the user list headlines to improve the width of the user list table

### Testing Instructions
- Go to user list
- Apply patch
- Headline for last visit and registration date are shorten

### Actual result BEFORE applying this Pull Request
Longer headline texts

grafik

### Expected result AFTER applying this Pull Request
Shorter headline text

grafik

### Additional information
I have not added it to #35285 because I'm not 100% sure about this one. Both PR make the user list "nicer".

### CODE REVIEW

**Comment by Kostelano** [comment]
~I have tested this item :white_check_mark: successfully on 9d7eaefd93bb7c8bf57d201e762185664e41de10~<br /><br />Sorry, sent a successful test to the wrong place. Delete please

**Comment by brianteeman** [comment]
As a general rule I am against changing the english to "fit" because it obviously doesn't resolve the issue in other languages.

In this specific case the english does still make sense so I would accept it as a "band aid" to the bigger problem.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 9d7eaefd93bb7c8bf57d201e762185664e41de10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35286">issues.joomla.org/tracker/joomla-cms/35286</a>.</sub>

File: administrator/language/en-GB/com_users.ini [hunk 1]
```diff
@@ -145,7 +145,7 @@ COM_USERS_HEADING_GROUPS="User Groups"
 COM_USERS_HEADING_GROUP_TITLE="Group Title"
 COM_USERS_HEADING_GROUP_TITLE_ASC="Group Title ascending"
 COM_USERS_HEADING_GROUP_TITLE_DESC="Group Title descending"
-COM_USERS_HEADING_LAST_VISIT_DATE="Last Visit Date"
+COM_USERS_HEADING_LAST_VISIT_DATE="Last Visit"
 COM_USERS_HEADING_LAST_VISIT_DATE_ASC="Last visit date ascending"
```
Comment (chmst) [inline administrator/language/en-GB/com_users.ini:149]: Shouldn't be the ordering be equal to the headline text?

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 9d7eaefd93bb7c8bf57d201e762185664e41de10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35286">issues.joomla.org/tracker/joomla-cms/35286</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35286">issues.joomla.org/tracker/joomla-cms/35286</a>.</sub>

**Comment by richard67** [comment]
Thanks.

**Pull request merged**